Header header

string[] bodies
float32[] distances
float32[] mutual_gazes
float32[] engagement_values
float32[] pose_estimation_confidences
uint8[] engagement_levels
float32[] engagement_level_confidences
uint8[] motion_activities
float32[] motion_activity_confidences
bool[] groups
bool[] group_with_robot
float32[] group_confidences
bool waiting
uint8 decision_action
string decision_target
